documentation/docs/15-configuration.md

```diff
@@ -154,6 +154,8 @@ When pages are prerendered, the CSP header is added via a `<meta http-equiv>` ta
 
 > When `mode` is `'auto'`, SvelteKit will use nonces for dynamically rendered pages and hashes for prerendered pages. Using nonces with prerendered pages is insecure and therefore forbidden.
 
+Note that most [Svelte transitions](https://svelte.dev/tutorial/transition) work by creating an inline `<style>` element. If you use these in your app, you must either leave the `style-src` directive unspecified or add `unsafe-inline`.
```

Comment:
should this be prefixed with `>` to make it more noticeable?

Reply:
If we only get to pick one, then I'd choose this new one :smile: Two call out boxes isn't bad though. I also think we've got one or two that are a big block with multiple paragraphs like:
```
>When...
>
>Note...
```

Reply:
yeah, multiple paragraphs is fine, though usually i think it's multiple paragraphs about a single concept, whereas these are two separate thoughts. will keep the two consecutive blockquotes 

```suggestion
> Note that most [Svelte transitions](https://svelte.dev/tutorial/transition) work by creating an inline `<style>` element. If you use these in your app, you must either leave the `style-src` directive unspecified or add `unsafe-inline`.
```